src/main/java/dev/walshy/sfmetrics/MetricsModule.java

```diff
@@ -0,0 +1,45 @@
+package dev.walshy.sfmetrics;
+
+import dev.walshy.sfmetrics.charts.AddonsChart;
+import dev.walshy.sfmetrics.charts.AutoUpdaterChart;
+import dev.walshy.sfmetrics.charts.CommandChart;
+import dev.walshy.sfmetrics.charts.CompatibilityModeChart;
+import dev.walshy.sfmetrics.charts.GuideLayoutChart;
+import dev.walshy.sfmetrics.charts.PlayerLanguageChart;
+import dev.walshy.sfmetrics.charts.ResearchesEnabledChart;
+import dev.walshy.sfmetrics.charts.ResourcePackChart;
+import dev.walshy.sfmetrics.charts.ServerLanguageChart;
+import dev.walshy.sfmetrics.charts.ServerSizeChart;
+import dev.walshy.sfmetrics.charts.SlimefunVersionChart;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import org.bstats.bukkit.Metrics;
+
+@SuppressWarnings("unused")
```

Comment:
why though?

Reply:
I hate IDE warnings for stuff I know is used.

Reply:
What even are you getting this warning for...?

Reply:
The start method of course.

Reply:
But it's `public static` :NotLikeThis:
You need to configure your IDE, man...